plugins/modules/grafana_folder.py

```diff
@@ -0,0 +1,277 @@
+#!/usr/bin/python
```

Comment:
Did you mean to add a new module as part of this refractor?

Reply:
no ... but I made the patch after the branch adding this module. I will have to merge this after https://github.com/ansible-collections/grafana/pull/10